CMakeLists.txt

```diff
@@ -0,0 +1,120 @@
+cmake_minimum_required (VERSION 2.6)
+project (LuaGumbo C)
+
+set(CMAKE_C_STANDARD_REQUIRED true)
+set(CMAKE_C_STANDARD 99)
+
+if(UNIX)
+  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+endif()
+
+# Lua
+if(WIN32)
+  find_package (Lua REQUIRED)
+else()
+  # Only the include dir matters on UNIX
```

Comment:
Doesn't this conflict with the installation rules you added in later commits?

Reply:
True. What really matters is that LUA_INCLUDE_DIR is valid (i.e. lua.h is found), as that implies the LUA_VERSION_* variables would have been set by find_package. It'd make more sense to check for the LUA_VERSION_ variables as well as LUA_INCLUDE_DIR than to add REQUIRED in my opinion.

Something like:
```
  if(NOT LUA_INCLUDE_DIR OR NOT LUA_VERSION_MAJOR OR NOT LUA_VERSION_MINOR)
    message( FATAL_ERROR "Path to lua.h is required, but could not be found" )
  endif()
```